resolwe/flow/routing.py

```diff
@@ -0,0 +1,25 @@
+""".. Ignore pydocstyle D400.
+
+=====================================
+Django Channels Routing Configuration
+=====================================
+
+"""
+
+from channels.routing import RouteClass
```

Comment:
The [Django Channels documentation](http://channels.readthedocs.io/en/stable) never mentions `RouteClass` directly.

I would prefer if you would use `from channels import route_class` as given in the [official documentation's example](http://channels.readthedocs.io/en/stable/generics.html?highlight=route_class#generic-consumers).